### PR TITLE
Artifact manager enhancement

### PR DESCRIPTION
**Pull request by hobbit1983**
Signed-off-by: Will Yates <hobbit1983@gmail.com>

### CODE REVIEW

**Comment by hobbit1983** [comment]
@Michael-Baylis this fixes your defect - can you buddy check the code
@CaroMac Can you review the updates to snippets please

**Comment by hobbit1983** [comment]
fixes galasa-dev/projectmanagement#246

File: changeLogs/changes-0.8.0.md [hunk 1]
```diff
@@ -5,3 +5,4 @@
 | 177  | zOS 3270 Terminal has changed InterruptedException to TerminalInterruptedException to make test catches easier |
 | JATconv | Handle READ BUFFER and rejected device selection |
 | 258 | Refactor zOS TSO and UNIX command managers |
+| 246 | New API in Artifact manager to make streaming text content a single step process |
```
Comment (Michael-Baylis) [inline changeLogs/changes-0.8.0.md:8]: new TPI.

Comment (hobbit1983) [reply changeLogs/changes-0.8.0.md]: Updated. thanks

**Pull request merged**